lib/src/Uncertainty/Distribution/Arcsine.cxx

```diff
@@ -43,13 +43,28 @@ Arcsine::Arcsine()
 }
 
 /* Parameters constructor */
+Arcsine::Arcsine(const NumericalScalar a,
+                 const NumericalScalar b)
+  : ContinuousDistribution()
+  , a_(a)
+  , b_(b)
+{
+  if (a >= b)
+    throw InvalidArgumentException(HERE) << "in Arcsine : a must be smaller than b";
```

Comment:
As already discussed, it would be better to negate these checks: if you want to ensure that a is smaller than b, then it should be written as
```
if (!(a < b))
    throw InvalidArgumentException(HERE) << "in Arcsine : a must be smaller than b";
```
This way, it catches all gotchas (a or b being NaN, etc).